README.md

```diff
@@ -30,11 +54,17 @@ async fn main() {
 SQL Query
 
 ```rust
-use spiceai::Client;
+use spiceai::ClientBuilder;
 
 #[tokio::main]
 async fn main() {
-  let mut client = Client::new("API_KEY").await.unwrap();
+  let mut client = ClientBuilder::new()
+    .with_api_key("API_KEY")
+    .with_spiceai_cloud()
```

Comment:
May be just `.with_spiceai_cloud(API_KEY)`?

Reply:
I think `api_key` should be separate - we might have it for local runtime as well